### DISCUSSION TITLE
Authorization succeeded, but it has been waiting

### DISCUSSION THREAD

**Discussion by li1243893836** [community/community#15719] [Copilot Conversations]
![image](https://user-images.githubusercontent.com/56986669/165694806-f3fb1aad-e482-4716-ad97-b7ae42b94c22.png)
![image](https://user-images.githubusercontent.com/56986669/165694838-c4b4ebca-19e4-45c8-b62e-362c16cf4b54.png)

The authorization was successful on GitHub, but on idea, he has been waiting, and this problem has been after updating the plug-in.

**Comment by sav7ng** [comment]
> ![image](https://user-images.githubusercontent.com/56986669/165694806-f3fb1aad-e482-4716-ad97-b7ae42b94c22.png) ![image](https://user-images.githubusercontent.com/56986669/165694838-c4b4ebca-19e4-45c8-b62e-362c16cf4b54.png)
> 
> The authorization was successful on GitHub, but on idea, he has been waiting, and this problem has been after updating the plug-in.

Me too, but I'll try the authorization step after I turn off all agents.

**Comment by davecheney** [comment]
Which version of VSCode are you using? Which version of the copilot extension are you using?
